pkg/util/test/util.go

```diff
@@ -4250,6 +4365,27 @@ func ValidateTelemetryV2Enabled(pxImageList map[string]string, cluster *corev1.S
 		return fmt.Errorf("failed to validate that Telemetry is Healthy in pxctl status, Err: %v", err)
 	}
 
+	// Validate registration certificate management via Container Orchestrator
+	ccmGoImage, ok := pxImageList["telemetry"]
+	if !ok {
+		return fmt.Errorf("failed to find image for telemetry")
+	}
+
+	ccmGoVersionStr := strings.Split(ccmGoImage, ":")[len(strings.Split(ccmGoImage, ":"))-1]
+	ccmGoVersion, err := version.NewSemver(ccmGoVersionStr)
+	if err != nil {
+		return fmt.Errorf("failed to find telemetry image version")
+	}
+
+	minimumPxVersionCO, _ := version.NewVersion("3.2")
+	minimumCcmGoVersionCO, _ := version.NewVersion("1.2.3")
+	pxVersion := GetPortworxVersion(cluster)
+	if pxVersion.GreaterThanOrEqual(minimumPxVersionCO) && ccmGoVersion.GreaterThanOrEqual(minimumCcmGoVersionCO) { // Validate the versions
```

Comment:
No PX Operator version dependency?

Reply:
I added one... at the time I was not sure we needed it but we do actually.